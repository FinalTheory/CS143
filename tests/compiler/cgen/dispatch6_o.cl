-- Tests dispatch on void

class A {
  f():Int {5};
};

class Main inherits IO {
  a:A;
  main():Int { a.f() };
};
