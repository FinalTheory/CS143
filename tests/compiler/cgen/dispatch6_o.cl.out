SPIM Version 6.5 of January 4, 2003
Copyright 1990-2003 by James R. Larus (larus@cs.wisc.edu).
All Rights Reserved.
See the file README for a full copyright notice.
Loaded: /usr/class/cs143/cool/lib/trap.handler
dispatch6_o.cl:1: Dispatch to void.
Stats -- #instructions : 151
         #reads : 23  #writes 22  #branches 27  #other 79
